worker/src/enclave/worker_api_direct_server.rs

```diff
@@ -210,66 +209,101 @@ pub fn handle_direct_invocation_request(req: DirectWsServerRequest) -> Result<()
 
 #[no_mangle]
 pub unsafe extern "C" fn ocall_update_status_event(
-    hash_encoded: *const u8,
-    hash_size: u32,
-    status_update_encoded: *const u8,
-    status_size: u32,
+    _hash_encoded: *const u8,
+    _hash_size: u32,
+    _status_update_encoded: *const u8,
+    _status_size: u32,
 ) -> sgx_status_t {
-    let mut status_update_slice =
-        slice::from_raw_parts(status_update_encoded, status_size as usize);
-    let status_update = TrustedOperationStatus::decode(&mut status_update_slice).unwrap();
-    let mut hash_slice = slice::from_raw_parts(hash_encoded, hash_size as usize);
-    if let Ok(hash) = Hash::decode(&mut hash_slice) {
-        // Aquire watched list lock
-        let mutex = load_watched_list().unwrap();
-        let mut watch_list = mutex.lock().unwrap();
-        let mut continue_watching = true;
-        if let Some(client_event) = watch_list.get_mut(&hash) {
-            let mut event = &mut client_event.response;
-            // Aquire result of old RpcResponse
-            let old_result: Vec<u8> = event.result.clone();
-            let mut result = RpcReturnValue::decode(&mut old_result.as_slice()).unwrap();
-
-            match status_update {
-                TrustedOperationStatus::Invalid
-                | TrustedOperationStatus::InSidechainBlock(_)
-                | TrustedOperationStatus::Finalized
-                | TrustedOperationStatus::Usurped => {
-                    // Stop watching
-                    result.do_watch = false;
-                    continue_watching = false;
+    // Not removing this function, as we may need this function later. So commented it out
```

Comment:
Keep old code in version tree not in comments.

Reply:
removed